timesketch/frontend/src/views/Home.vue

```diff
@@ -62,15 +62,37 @@ limitations under the License.
 
     <div v-if="mySketches.length && !search" class="section">
       <div class="container">
-        <p class="subtitle">My sketches</p>
-        <div class="card">
-          <div class="card-content">
-            <ts-sketch-list :sketches="mySketches"></ts-sketch-list>
-          </div>
-        </div>
+        <b-tabs v-model="activeTab">
+
+          <b-tab-item label="My sketches" :disabled="!mySketches.length">
+              <div class="card">
+                <div class="card-content">
+                  <ts-sketch-list :sketches="mySketches"></ts-sketch-list>
+                </div>
+              </div>
+            </b-tab-item>
+
+            <b-tab-item label="Shared with me" :disabled="!sharedSketches.length">
+              <div class="card">
+                <div class="card-content">
+                  <ts-sketch-list :sketches="sharedSketches"></ts-sketch-list>
+                </div>
+              </div>
+            </b-tab-item>
+
+            <b-tab-item label="Archived" :disabled="!myArchivedSketches.length">
+              <div class="card">
+                <div class="card-content">
+                  <ts-sketch-list :sketches="myArchivedSketches"></ts-sketch-list>
+                </div>
+              </div>
+            </b-tab-item>
+
+        </b-tabs>
       </div>
     </div>
 
+    <!--
```

Comment:
Why do you want to keep that as a comment?

Reply:
good catch, removed now.